stresstest.c

```diff
@@ -83,15 +83,15 @@ int main(int argc, char *argv[]) {
             size_t send_pos = 0;
             while (buffer_pos - send_pos >= 16) {
                 const size_t bytes_to_send = 1 + rand() % 16;  // 1..16
-                write(STDOUT_FILENO, buffer + send_pos, bytes_to_send);
+                (void)! write(STDOUT_FILENO, buffer + send_pos, bytes_to_send);
                 usleep(50);  // let ttyplot read this before proceeding
                 send_pos += bytes_to_send;
```

Comment:
The compiler is telling us that we should check the return value of `write()`. Maybe we should listen to it instead of trying to silence it.

```suggestion
                const ssize_t bytes_sent = write(STDOUT_FILENO, buffer + send_pos, bytes_to_send);
                usleep(50);  // let ttyplot read this before proceeding
                if (bytes_sent > 0)
                    send_pos += bytes_sent;
```